Wrappers/Python/ccpi/optimisation/functions/MixedL21Norm.py

```diff
@@ -62,25 +48,9 @@ def __call__(self, x):
         """
         if not isinstance(x, BlockDataContainer):
             raise ValueError('__call__ expected BlockDataContainer, got {}'.format(type(x))) 
+              
+        return x.pnorm(p=2).sum()                                
```

Comment:
Simpler to use `norm`? This function is specifically for 2-norm, so pnorm doesn't seem to be needed?

Reply:
Here, we have block data containers and want to compute `np.sum(np.sqrt(x1**2 + x2**2))`